shared/xccdf/services/nfs.xml

```diff
@@ -86,17 +86,17 @@ this service should be disabled.
 
 </Group> <!-- end disabling_nfs -->
 
-<Group id="nfs_configuring_all_machines">
+<Group id="nfs_configuring_all_systems">
```

Comment:
I noticed that I missed some "Machine" occurrences in this Group.

Breaking existing tailoring files is something I didn't consider when making this change, so I'll revert it.
And I guess users rely much more on title and description of Rule/Group than its ID, so as long as title and description are in good shape users should be safe from misunderstanding it.